.github/workflows/dockerhub-cleanup.yaml

```diff
@@ -17,10 +17,20 @@ jobs:
     # The type of runner that the job will run on
     runs-on: ubuntu-20.04
 
+    # We need `jq` for the script, so we run in a container with it installed
+    container:
+      image: dwdraju:alpine-curl-jq
+
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      - name: Delete image
+      # Checks-out the repo under $GITHUB_WORKSPACE so we can access the delete script
+      - name: Checkout Code
+        uses: actions/checkout@v2
```

Comment:
I'm not super familiar with GHA, but will this action run inside the `alpine-curl-jq`? It wont have `git` installed if so.

Can we make this workflow trigger on push's too this branch to test?

Reply:
crap. Good point. Let's see what happens

Reply:
Looks like the checkout isn't an issue. I have screwed up the `TAG` env var though